drivers/net/rpmsgdrv.c

```diff
@@ -742,19 +742,6 @@ static int net_rpmsg_drv_ifup(FAR struct net_driver_s *dev)
   net_ipv6addr_copy(msg.ipv6netmask, dev->d_ipv6netmask);
 #endif
 
-  /* Send the message */
```

Comment:
Isn't this is related to previous; this is needed if arp_out has been sent, so instead of removing, move it behind !IFF_IS_NOARP?

Reply:
This just disables sending of NET_RPMSG_IFUP message on startup. This feature is not supported on Linux side and this driver get stuck in here if this is enabled. Probably this should be just commented out..

Reply:
Did this more polished way. All other than TRANSFER commands are now disabled in send_recv().
We can remove this if we add support in Linux side.